app/ide-desktop/lib/dashboard/esbuild-config.ts

```diff
@@ -96,6 +104,31 @@ function esbuildPluginGenerateTailwind(): esbuild.Plugin {
     }
 }
 
+/** Plugin that packages the ReactToastify.css. */
+export function esbuildPluginAddReactifyCSS(): esbuild.Plugin {
```

Comment:
you might want to explore using a css `@import` so that we only need the one `tailwind.css` - could probably rename it to something else if it's becoming a misnomer

Reply:
Brilliant suggestion! This makes everything so much simpler and cleaner!
(sorry, I have very little knowledge in CSS/webdev)